tty/pc-tty/ttypc_kbd.c

```diff
@@ -16,14 +16,45 @@
 #include <stdint.h>
 #include <string.h>
 #include <unistd.h>
+#include <stdbool.h>
 
+#include <sys/file.h>
 #include <sys/interrupt.h>
 #include <sys/io.h>
 #include <sys/threads.h>
 #include <sys/reboot.h>
 
+#include <libklog.h>
+#include <fcntl.h>
+#include <posix/utils.h>
+
 #include "ttypc_kbd.h"
+#include "ttypc_mouse.h"
 #include "ttypc_vga.h"
+#include "ttypc_ps2.h"
+#include "event_queue.h"
+
+
+#define CTL_THREAD_PRIORITY  1
```

Comment:
Please avoid using priority 1 if possible. Choose the highest value that still works.

Reply:
Priority 2 can already make the input sluggish. Also, the CTL_THREAD is responsible for capturing input for the console. I think that these threads should be left with priority 1 and process incoming input events as quickly as possible. They won't starve the rest of the system as the input event stream rate is reasonably limited in both kbd and mouse